SIDFactoryII/config.ini

```diff
@@ -33,6 +33,8 @@ Sound.Emulation.Resample            = 1         // If this is set to 1, the SID
 
 Sound.Output.Gain                   = 1.0       // Boost/lower volume. Sound can become distorted for value higher than 1.0.
 
+Sound.Emulation.SampleFrequency		= 44100     // Output sample frequency in Hz (22050/44100/48000/96000 etc.)
```

Comment:
Should we just make 48000 the default regardless of platform? @TheRealLaxity?

Reply:
Good question. I know that Windows uses 48Khz on most devices but have no idea how the other platforms are handling it, so 

Additionally I checked the SDL headers if you can query the audio device's preferred sample rate, but it seems you can't.

Reply:
Could we use this? https://wiki.libsdl.org/SDL_GetAudioDeviceSpec
With the purpose of automatically selecting the device's sampling rate when not overridden bij the user.

Reply:
I did a little test:

```c++
SDL_AudioSpec spec;
SDL_memset(&spec, 0, sizeof(spec));

const int specStatus = SDL_GetAudioDeviceSpec(0, 0, &spec);

Utility::Logging::instance().Info("Audio device frequency: %d", spec.freq);
```

And it logs `48000`. The requested freq is hardcoded in SF2 to 44100, but the device is reporting 48000.

Reply:
I just wanted to answer, and suddenly you merged the PR.. :)

Yeah, we could. Actually in the latest SDL there's https://wiki.libsdl.org/SDL_GetDefaultAudioInfo that doesn't need a device id, so that would be preferable.